File: Difficulty: Easy/Pattern 17/pattern-17.java
class Solution {

    void printTriangle(int n) {
        // code here
        
        for(int i=0;i<n;i++){
            int start=65;
                for(int space=0;space<n-i-1;space++){
                    System.out.print(" ");
                }   
                for(int j=0;j<=i;j++){
                    System.out.print((char)(start+j));
                    
                }
                for(int j=i-1;j>=0;j--){
                    System.out.print((char)(start+j));
                }
                System.out.println();
       }
    }
}

File: Difficulty: Easy/Pattern 17/pattern-17.py
#User function Template for python3

class Solution:
    def printTriangle(self, N):
        # Code here
        for i in range(1,N+1):
            start=65
            for space in range(N-i):
                print(" ",end="")
            for j in range(i):
                print(chr(start+j),end="")
            for k in range(i-1,0,-1):
                print(chr(start+k-1),end="")
            print()